db-migrations/000005_adding_owners.up.sql

```diff
@@ -0,0 +1,23 @@
+-- schema changes for account owners and champions
+BEGIN;
+
+create table person(
+    id serial primary key,
+    login varchar unique not null,
+    email varchar unique not null,
+    name varchar not null,
+    valid boolean not null
+)
+
+create table champions(
+    foreign key(people_id) references person (id),
+    foreign key(aws_account_id) references aws_account (id),
+    constraint champ_unique unique(people_id, aws_account_id)
```

Comment:
trying to use a constraint here as per https://stackoverflow.com/questions/10468657/postgres-unique-multi-column-index-for-join-table to guarantee we get unique pairs of people and accounts